api/control/v1alpha1/types.go

```diff
@@ -443,6 +457,42 @@ const (
 // Type checking errors are posted to the status.Errors of the Binding and BindingPolicy.
 type Expression string
 
+// Augmentation defines what is implicitly queryable in the CEL expressions.
+// This means that while the listed fields are not part of the object's schema,
+// they can be referred to in the CEL expressions.
+//
+// Besides for Augmentations, the other objects queryable in CEL expressions are:
```

Comment:
Rather than define Augmentation and also document two things "besides Augmentation", I think it would be simpler to just have one level of documentation that defines all four top-level variables usable in the expressions.

Reply:
This is moot at the moment, since the full description has been removed from type.go. However, I think that types.go should have a full description.